tests/wiTest.cpp

```diff
@@ -1689,11 +1688,24 @@ TEST_CASE("WindowsInternalTests::HandleWrappers", "[resource][unique_any]")
 
 #if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
     {
-        auto verify = MakeSecureDeleterMallocSpy();
-        REQUIRE_SUCCEEDED(::CoRegisterMallocSpy(verify.Get()));
-        auto removeSpy = wil::scope_exit([&] {
-            ::CoRevokeMallocSpy();
-        });
+        witest::detoured_thread_function<&::CoTaskMemFree> detour;
+        REQUIRE_SUCCEEDED(detour.reset([](void* p) {
+            if (p != nullptr)
+            {
+                IMalloc* pMalloc = nullptr;

```

Comment:
`wil::com_ptr<IMalloc>`?

Reply:
Fair. Probably shouldn't have blindly copy pasted code without giving it a once-over